engine/src/Texture.cpp

```diff
@@ -33,3 +33,5 @@ namespace engine::resources {
         }
     }
 }
+
+
```

Comment:
[STYLE] Bilo bi korisno obrisati suvišne prazne linije kako bi se smanjio šum prilikom pregleda promenjenih fajlova i održala konzistentnost koda.

Ukoliko imate instaliran GitToolBox prema upustvu sa projekta moći ćete da vidite sve promene nad fajlovima preko `Git -> Commit` opcije iz menija pre nego što uradite commit.